### ISSUE TITLE
nx commands: affected:lint inserts weird arguments/doesn't support arguments

### ISSUE THREAD

**Issue by Hotell** [nrwl/nx#2721]
# Prerequisites

- [x] I am running the latest version
- [x] I checked the documentation (nx.dev) and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (not related to React, Angular or any dependency)

## Expected Behavior

- allow passing arguments to underlying lint builder

**WHY?**
- We are using danger, which consumes json output from linter. currently we cannot leverage `nx` tooling for pipeline checks as we need to provide json eslint output for danger.
- also when `--format=json` is used it inserts weird second argument `--format:write=json`

## Current Behavior

![image](https://user-images.githubusercontent.com/1223799/77543341-d75cd300-6ea7-11ea-9c29-5bd1963d69cb.png)

### Context
```
  @nrwl/angular : Not Found
  @nrwl/cli : 9.1.2
  @nrwl/cypress : 9.1.2
  @nrwl/eslint-plugin-nx : 9.1.2
  @nrwl/express : Not Found
  @nrwl/jest : 9.1.2
  @nrwl/linter : 9.1.2
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 9.1.2
  @nrwl/react : 9.1.2
  @nrwl/schematics : Not Found
  @nrwl/tao : 9.1.2
  @nrwl/web : 9.1.2
  @nrwl/workspace : 9.1.2
  typescript : 3.8.3
```

**Comment by FrozenPandaz** [comment]
Commands that are intended for the builder themselves should be passed in after a delimiting `--`.

Can you try the following command please?

```bash
yarn nx affected:lint -- --format json
```

**Comment by Hotell** [comment]
still same weird error:

![image](https://user-images.githubusercontent.com/1223799/77775840-69eaa700-704d-11ea-92dd-4ea7e7c075ff.png)

`yarn nx -- affected:lint -- --format json`

your proposed command throws following error:

![image](https://user-images.githubusercontent.com/1223799/77776052-b7ffaa80-704d-11ea-92d3-8ca9d467d2ef.png)

**Comment by Hotell** [comment]
but following works:

`$ yarn nx affected:lint -- --base master -- --format json`

![image](https://user-images.githubusercontent.com/1223799/77776335-20e72280-704e-11ea-933f-3707ec5c16ae.png)

**Comment by bekos** [comment]
@Hotell This seemed to be a issue with the alias of `nx format` command. It should be already fixed via ebe49c79175292c14e4d5849f4257026997ccc8c. Please try it and reopen if this is still an issue.
